setup.cfg

```diff
@@ -1,80 +1,9 @@
-[metadata]
-name = napari-imagej
-version = 0.1.0.dev0
-author = ImageJ2 developers
-author_email = ctrueden@wisc.edu
-url = https://github.com/imagej/napari-imagej
-license = BSD-3-Clause
-description = Use ImageJ functionality from napari
-long_description = file: README.md
-long_description_content_type = text/markdown
-include_package_data = True
-classifiers =
-    Development Status :: 2 - Pre-Alpha
-    Intended Audience :: Developers
-    Framework :: napari
-    Topic :: Software Development :: Testing
-    Programming Language :: Python
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-    Operating System :: MacOS
-    Operating System :: Microsoft :: Windows
-    Operating System :: Unix
-    License :: OSI Approved :: BSD License
-project_urls =
-    Bug Tracker = https://github.com/imagej/napari-imagej/issues
-    Documentation = https://github.com/imagej/napari-imagej#README.md
-    Source Code = https://github.com/imagej/napari-imagej
-    User Support = https://github.com/imagej/napari-imagej/issues
-
-[options]
-packages = find:
-python_requires = >=3.7
-package_dir =
-    =src
-
-# add your package requirements here
-install_requires =
-    confuse
-    labeling >= 0.1.12
-    # TODO: remove upon next napari release
-    imageio != 2.22.1
-    napari
-    numpy
-    magicgui >= 0.5.1
-    pyimagej >= 1.3.0
-    scyjava >= 1.8.1
-
-[options.packages.find]
-where = src
-
-[options.package_data]
-napari-imagej = napari.yml
-
-[options.entry_points] 
-napari.manifest = 
-    napari-imagej = napari_imagej:napari.yml
-
-[options.extras_require]
-
-# Ensure any changes to this list are also added to environment-test.yml!
-dev =
-    autopep8
-    black
-    build
-    flake8
-    isort
-    pyqt5
-    pytest
-    pytest-cov
-    pytest-env
-    pytest-qt
-    qtpy
-    numpy
-
 [flake8]
 # See https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8
 max-line-length = 88
-extend-ignore = E203
\ No newline at end of file
+exclude =
+    bin
+    build
+    docs
+    dist
+extend-ignore = E203
```

Comment:
The scyjava project has the following additional lines:
```
# Unfortunately, flake8 doesn't pick up on our declaration in pyproject.toml
min_python_version = 3.7.0
```
But pyimagej and jgo do not have it.

Reply:
Yeah, @elevans and I have been dealing with that for a while.

One thing that `flake8` does is tell you when you're using features that are newer than your minimum python version. For that reason, you have to tell flake8 what your minimum python version is. Unfortunately, `pyproject.toml` configuration of `flake8` is a [very heated topic](https://github.com/PyCQA/flake8/issues/234). It's still not supported.

Looking around now, though, I found a flake8 plugin, [`Flake8-pyproject`](https://pypi.org/project/Flake8-pyproject/), that replaces the `setup.cfg` parser with a `pyproject.toml` parser. I'm going to try to use that, in which case we don't need that line or `setup.cfg` anymore.

Reply:
Alright, I enabled `flake8-pyproject`, seems to be working great! No more `setup.cfg`!

Reply:
There is also https://pypi.org/project/flake9/

Reply:
Well, the nice thing about `flake8-pyproject` is that it's a plugin, not a fork.

Also see https://github.com/john-hen/Flake8-pyproject/issues/2, where they talk about the reasons for this project.